tethys_cli/cli_helpers.py

```diff
@@ -39,6 +40,12 @@ def get_manage_path(args):
     return manage_path
 
 
+def get_django_manage_commands():
+    manage_commands = sorted(list(get_commands().keys()))
+
+    return manage_commands
+
+
```

Comment:
This function isn't generic enough to be here. Move it to the `manage_commands.py`

Reply:
removed.